### ISSUE TITLE
Highlighter should throw an exception if an unknown language is requested

### ISSUE THREAD

**Issue by marijnvdwerf** [scrivo/highlight.php#7]
**Input**
```
$hl = new Highlighter();
$r = $hl->highlight(':+1:', '<p>User-supplied content</p>');
```

**Expected**
(Something like) `/Highlight/UnkownLanguageException` being thrown.

**Actual**
```
Notice: Undefined index: :+1:
```

**Comment by scrivo** [comment]
Ok, but that will be a DomainException then.
http://php.net/manual/en/class.domainexception.php

**Comment by scrivo** [comment]
This issues was addressed in release 8.6. Thanks for reporting.
